src/github/pullRequestOverview.ts

```diff
@@ -563,6 +568,78 @@ export class PullRequestOverviewPanel {
 		});
 	}
 
+	private async deleteBranch(message: IRequestMessage<any>) {
+		const branchInfo = await this._pullRequestManager.getBranchNameForPullRequest(this._pullRequest);
+		let actions: (vscode.QuickPickItem & { type: 'upstream' | 'local' | 'remote' })[] = [];
+		let branchHeadRef = this._pullRequest.head!.ref;
+
+		actions.push({
+			label: `Delete remote branch ${this._pullRequest.remote.remoteName}/${branchHeadRef}`,
+			description: `${this._pullRequest.remote.normalizedHost}/${this._pullRequest.remote.owner}/${this._pullRequest.remote.repositoryName}`,
+			type: 'upstream',
+			picked: true
+		});
+
+		if (branchInfo) {
+			actions.push({
+				label: `Delete local branch ${branchInfo.branch}`,
+				type: 'local',
+				picked: !!Storage.getPreference('ghpr.deleteBranch.shouldSelectLocalBranch')
```

Comment:
I think it would be better to just make this and the remote one explicit settings, so that they're easier to configure